### DISCUSSION TITLE
Any examples to listen to anywhere?

### DISCUSSION THREAD

**Discussion by mdkberry** [stakira/OpenUtau#771] [General]
Seems weird that not a single audio example exists that I can find on this github. What does it sound like?

I am specifically interested in writing lyrics for an english male or female vocal and then have it play from using midi in my DAW. which it seems to be able to do, but what do the vocals sound like?

might get more traction if there were some examples so we know what it can do.

**Comment by pseudo-producer** [comment]
I didn't understand your question. Are you looking for an English voice bank? If that's the case, any English Utau would do. As for "playing in your DAW", would it be like what some people do in Reaper, by importing a voice bank and using it in the program as if it were a soundfont or .sf/.sfx file?

**Comment by oxygen-dioxide** [comment]
> 

Openutau doesn't ship with a default voicebank, but anyone can make a voicebank for openutau. You can search "openutau" on video and music websites to find demos and voicebank for it.

**Reply by pseudo-producer** [reply]
I take advantage of your comment to ask a question based on the creation of voice banks. How is it possible to create a voice bank in Open Utau if you need to install the voice banks to use the software?
